packages/itwinui-css/src/input/classes.scss

```diff
@@ -21,3 +25,33 @@
 .iui-end-icon {
   @include iui-end-icon;
 }
+
+.iui-input-flex-container {
+  @include iui-input-flex-container;
+
+  > input[type='search'] {
+    @include iui-search-box;
+  }
+
+  &[data-iui-size='small'] {
+    @include iui-input-size(small);
+  }
+
+  &[data-iui-size='large'] {
+    @include iui-input-size(large);
+  }
+
+  @each $status in positive, warning, negative {
+    &.iui-#{$status} {
+      @include iui-input-status($status: $status, $iconSelector: 'svg', $textSelector: 'input');
+    }
+  }
+}
+
+.iui-invisible-borders {
+  @include iui-invisible-borders;
```

Comment:
this should be autoapplied on input when used in container. that will make it possible for user to pass a regular `input`

Reply:
Including in input.